views/components/navbar.ejs

```diff
@@ -6,19 +6,19 @@
         <div class="menu">
             <hr />
             <li class="navbar-item">
-                <a href="#about">ABOUT</a>
+                <a href="/#about">ABOUT</a>
```

Comment:
I'm not sure what you meant by a root parameter but I'm pretty sure this solves the issue